lib/logflare/backends/adaptor/postgres_adaptor/repo/migrations/add_log_events.ex

```diff
@@ -6,9 +6,10 @@ defmodule Logflare.Backends.Adaptor.PostgresAdaptor.Repo.Migrations.AddLogEvents
 
   alias Logflare.Backends.Adaptor.PostgresAdaptor.PgRepo
 
-  def generate_migration(source_backend) do
+  def generate_migration(%{source: %{token: token}} = source_backend) do
```

Comment:
can we add docstring for this, i think we missed it in initial implementaiton. should mention that source needs to be preloaded.